lib/twilio_adapter.rb

```diff
@@ -0,0 +1,15 @@
+class TwilioAdapter
+  attr_reader :client, :twilio_number
+  def initialize
+    @client = Twilio::REST::Client.new(ENV['TWILIO_SID'], ENV['TWILIO_AUTH'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.